packages/lib/services/e2ee/NativeEncryption.node.ts

```diff
@@ -0,0 +1,35 @@
+import { NativeEncryptionInterface } from './types';
+import { promisify } from 'util';
+import crypto = require('crypto');
+
+class NativeEncryption implements NativeEncryptionInterface {
```

Comment:
I'm not sure the use of a class makes sense here. Is it just to group these functions together? In that case a file `crypto.ts` might be preferable with plain functions inside.

Also the fact that it is "native" doesn't really matter, so it should not appear in the name.

Reply:
> Also the fact that it is "native" doesn't really matter, so it should not appear in the name.

I will remove it.

> Is it just to group these functions together? In that case a file `crypto.ts` might be preferable with plain functions inside.

I guess this is the grouped plain functions, right?
https://github.com/laurent22/joplin/blob/v3.0.1/packages/lib/services/e2ee/RSA.node.ts#L11

Reply:
I have renamed these modules to `Crypto` in commit a2be9f1126501561cc104e02ae91f03ac4a7dc5c
However, I just noticed that the name `Crypto` has been used in the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Crypto). I guess it's better to use a different name.

Reply:
I have removed the class definition in commit 17aa0fdac7cc2a83a574d96612db0b953dae2c47.

Reply:
As @tessus suggested, I've replaced the name to `Krypto`